File: check_braces.py
with open('src/components/PageBlocks.tsx', 'r', encoding='utf-8') as f:
    text = f.read()

count = 0
for char in text:
    if char == '{': count += 1
    elif char == '}': count -= 1
print(f"Final brace balance: {count}")

File: check_braces.js
import fs from 'fs';
let pb = fs.readFileSync('src/components/PageBlocks.tsx', 'utf8');

let depth = 0;
for (let i=0; i<pb.length; i++) {
    if (pb[i] === '{') depth++;
    if (pb[i] === '}') depth--;
}
console.log("Final depth:", depth);
